libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -231,6 +233,86 @@ describe('ModalWrapperComponent', () => {
     });
   });
 
+  describe('with embedded page progress component', () => {
+    describe('with static page progress', () => {
+      beforeEach(() => {
+        spectator = modalWrapperTestBuilder
+          .flavor('modal')
+          .withStaticPageProgress()
+          .build();
+        spectator.detectComponentChanges();
+      });
+
+      afterEach(() => {
+        // Ensure any observers are destroyed:
+        spectator.fixture.destroy();
+      });
+
+      it('should move embedded page progress content to ion toolbar', () => {
```

Comment:
Consider aligning with existing test descriptions:
```suggestion
      it('should move embedded page progress to wrapper component', () => {
```